src/field-element-creation-strategy.ts

```diff
@@ -15,35 +16,43 @@ export class FieldElementCreationStrategy implements FormElementCreationStrategy
             this.createInputFields(form, element);
         }
     }
-
+    
     private createPhoneInput = async (form: HTMLFormElement, element: any) => {
         const paragraph = this.elementCreator.createElement('div');
-        paragraph.textContent = element.name;
-        form.appendChild(paragraph);
-        const select = this.elementCreator.createElement('select');
-        const input = this.elementCreator.createElement('input') as HTMLInputElement;
-        this.setInputAttributes(input, element);
-
-        const countryTelMap = await this.fetchCountryPhoneCodes();
-
-        countryTelMap.forEach((diallingCode: string, countryName: string) => {
-            const option = this.elementCreator.createElement('option') as HTMLOptionElement;
-            option.text = `${countryName}`;
-            option.value = `${diallingCode}`;
-            select.appendChild(option);
-        });
-
-        form.appendChild(select);
-        form.appendChild(input);
+        
+        if (paragraph) {
+            paragraph.textContent = element.name;
+            form.appendChild(paragraph);
+    
+            const select = this.elementCreator.createElement('select');
+            if (select) {
+                const input = this.elementCreator.createElement('input') as HTMLInputElement;
+                this.setInputAttributes(input, element);
+    
+                const countryTelMap = await this.fetchCountryPhoneCodes();
+    
+                countryTelMap.forEach((diallingCode: string, countryName: string) => {
+                    const option = this.elementCreator.createElement('option') as HTMLOptionElement;
+                    option.text = `${countryName}`;
+                    option.value = `${diallingCode}`;
+                    select.appendChild(option);
+                });
```

Comment:
ten kawalek to bym wydzielil do innej metody, nie ma potrzeby rozwlekac tej metody na tak wiele linii a ten kawalek moze byc oddzielna metoda, po prostu zawolasz metode getCountryTelMap i tyle